functions/src/utils.js

```diff
@@ -131,10 +131,11 @@ function flattenDocument(obj, prefix = "") {
 
 /**
  * @param {DocumentSnapshot} firestoreDocumentSnapshot
+ * @param {Object} contextParams
  * @param {Array} fieldsToExtract
  * @return {Object} typesenseDocument
  */
-exports.typesenseDocumentFromSnapshot = async (firestoreDocumentSnapshot, fieldsToExtract = config.firestoreCollectionFields) => {
+exports.typesenseDocumentFromSnapshot = async (firestoreDocumentSnapshot, contextParams = {}, fieldsToExtract = config.firestoreCollectionFields) => {
```

Comment:
Since this was given a default value of an empty object, was it necessary to pass an empty object to calls to this function? e.g.:
```js
        const result = await typesenseDocumentFromSnapshot(documentSnapshot, {});
```

Reply:
good point. will update all call sites.

Reply:
updated the single callsite you pointed out test/utilsWithFlattening.spec.js line 20. Didn't see any other call sites.